geemap/geemap.py

```diff
@@ -2436,223 +2438,6 @@ def close_btn_clicked(b):
 
             return vis_widget
 
-    def _point_info(self, latlon, decimals=3, return_node=False):
-        """Create the ipytree widget for displaying the mouse clicking info.
-
-        Args:
-            latlon (list | tuple): The coordinates (lat, lon) of the point.
-            decimals (int, optional): Number of decimals to round the coordinates to. Defaults to 3.
-            return_node (bool, optional): If True, return the ipytree node.
-                Otherwise, return the ipytree tree widget. Defaults to False.
-
-        Returns:
-            ipytree.Node | ipytree.Tree: The ipytree node or tree widget.
-        """
-        from ipytree import Node, Tree
-
-        point_nodes = [
-            Node(f"Longitude: {latlon[1]}"),
-            Node(f"Latitude: {latlon[0]}"),
-            Node(f"Zoom Level: {self.zoom}"),
-            Node(f"Scale (approx. m/px): {self.get_scale()}"),
-        ]
-        label = f"Point ({latlon[1]:.{decimals}f}, {latlon[0]:.{decimals}f}) at {int(self.get_scale())}m/px"
-        root_node = Node(
-            label, nodes=point_nodes, icon="map", opened=self._expand_point
-        )
-
-        root_node.open_icon = "plus-square"
-        root_node.open_icon_style = "success"
-        root_node.close_icon = "minus-square"
-        root_node.close_icon_style = "info"
-
-        if return_node:
-            return root_node
-        else:
-            return Tree(nodes=[root_node])
-
-    def _pixels_info(

```

Comment:
Similarly, the refactored `_pixels_info()` function only has the `latlon` parameter. It also causes issues when clicking at locations without pixels.